packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -221,7 +229,10 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
                     title={this.bindings.i18n.t('this-answer-was-not-helpful')}
                     variant="dislike"
                     active={this.generatedAnswerState.disliked}
-                    onClick={this.generatedAnswer.dislike}
+                    onClick={() => {
```

Comment:
Out of curiosity, can you elaborate on why the suggested approach is better?

Reply:
Yeah basically it and that it could maybe cause unnecessary rerenders if it is for a prop given the function instance is a new one on every render. I honestly don't know the real impact this has. I found it because in a personal project using stencil I'm using the recommended eslint rules and this one came up https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#es6-classes